### ISSUE TITLE
[Runtypes] Example in Readme doesn't work

### ISSUE THREAD

**Issue by sstur** [johngeorgewright/runtyping#332]
The "Use from a script" example in the readme [here](https://github.com/johngeorgewright/runtyping/blob/071fcc2/packages/runtypes/README.md)
<img width="612" alt="image" src="https://user-images.githubusercontent.com/369384/180049780-bce0dd4c-65da-4f98-9e9f-ba455904fdf6.png">

Produces the following TypeScript error:
```
Argument of type '{ targetFile: string; }' is not assignable to parameter of type 'GeneratorOptions'.
  Type '{ targetFile: string; }' is not assignable to type '{ typeWriters: TypeWriters; module: string; targetFile: string; project?: Project; } & { runtypeFormat?: string; typeFormat?: string; }'.
    Type '{ targetFile: string; }' is missing the following properties from type '{ typeWriters: TypeWriters; module: string; targetFile: string; project?: Project; }': typeWriters, module ts(2345)
```

As can be seen here:
<img width="617" alt="image" src="https://user-images.githubusercontent.com/369384/180050046-faa9b04d-819e-4a8f-ae4c-5fe8a6f7b81c.png">

## Workaround

I was able to work around this by importing `RuntypesTypeWriters` directly from the internals of the package, but this seems hacky. Is there a better solution?

<img width="922" alt="image" src="https://user-images.githubusercontent.com/369384/180054181-ec4f4d17-8112-43d0-8d01-47fadbc9d6f7.png">

**Comment by johngeorgewright** [comment]
Ah yes. I forgot to update this after the move to a mono-repo. I'll update this shortly.

**Comment by johngeorgewright** [comment]
Fixed in `@runtyping/runtypes-v1.1.1`
